duneanaobj/StandardRecord/SRNDTrackMatch.h

```diff
@@ -0,0 +1,19 @@
+/// \author  J. Wolcott <jwolcott@fnal.gov> & F. Akbar <fakbar@ur.rochester.edu>
+/// \date    Nov. 2021
+
+#ifndef DUNEANAOBJ_SRNDTRACKMATCH_H
+#define DUNEANAOBJ_SRNDTRACKMATCH_H
+
+namespace caf
+{
+  class SRNDTrackMatch
+  {
+    public:
+      int larTrkIdx       = -1;
+      int tmsTrkIdx       = -1;
+      float xverseDispl   = -999.;
+      float angularDsipl  = -999.;
```

Comment:
I would prefer to keep to the all-lowercase philosophy here.

The prior art in the (sadly long-stalled) garsoft CAF branch would call this whole thing `SRNDTrackAssn` and the fields `larid` and `tmsid`
https://cdcvs.fnal.gov/redmine/projects/garsoft/repository/entry/StandardRecord/SRVertexAssn.h?rev=feature%2Fcaf_structure

Reply:
I'd also prefer a more user-friendly structure long term (declare one type of track "primary" and have the others hanging off their matches? Have a combined lar+tms track type?), but I understand this is how the underlying software thinks, so it's fine for now. Another place to try and synchronize with whatever the garsoft cafs do.

Reply:
switched to lowercase, at least (d0cf7448).  I guess in the long run we'd want ND-LAr tracks to the be the "primary" ones, but I don't want to make that call today, before we even have done any work with them to figure out what a workflow should look like, so I'm going to leave the rest until later.

Reply:
oh, I _will_ implement a name switch to `NDTrackAssn` though (that's a lot easier now than later), but I'll do that in a separate commit.

Reply:
implemented in 4eb84e9d